src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -73,6 +75,12 @@
     private int max;
     /** Package of the file we check. */
     private String packageName;
+    /** User-configured regular expression to ignore classes. */
+    private String excludeClassesRegexp = "";
+    /**
+     * Regular expression pattern compiled from excludeClassesRegexp variable.
+     */
+    private Pattern excludeClassesRegexpPattern = CommonUtils.createPattern(excludeClassesRegexp);
```

Comment:
@romani 
Are we OK to update both ClassDataAbstractionCouplingCheck and ClassFanOutComplexityCheck ? The initial issue says that update should be done only for ClassDataAbstractionCouplingCheck. Both checks extend AbstractClassCouplingCheck.